src/Elastic.Apm/Api/ISpan.cs

```diff
@@ -23,17 +23,17 @@ public interface ISpan
 		/// <summary>
 		/// The id of the span.
 		/// </summary>
-		int Id { get; set; }
+		string Id { get; set; }
 
 		/// <summary>
 		/// The name of the span.
 		/// </summary>
 		string Name { get; set; }
 
 		/// <summary>
-		/// Offset relative to the transaction's timestamp identifying the start of the span, in milliseconds.
+		/// Hex encoded 64 random bits ID of the parent transaction or span.
```

Comment:
Yes, that's the `ParentId`, but this applies to all ids: The ids are 64 (or 128 in case of `TraceId`) numbers. These numbers are hex-encoded into a `string`, the `string` is the hex representation of this number.  ...so not the `string` is hex-encoded, but the number into a `string`.